src/test/resources/data/xml/input/models.par

```diff
@@ -0,0 +1,281 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is is file useful ? already in resources in  src/test/resources/data/ieee14/_01/input/models.par ?

Reply:
The same name but the content may be different. It is used for testing the xml merge tool. In my opinion, it is better to separate test data.. So I will keep this file..

Anyway, the merge tool will be removed when the multi input-files implementation at dynawo finished

Reply:
ok